OnePassword.NET/Items/ItemBase.cs

```diff
@@ -28,6 +28,13 @@ public string Title
     [JsonPropertyName("category")]
     public Category Category { get; internal init; } = Category.Unknown;
 
+    /// <summary>
+    /// The item category ID.
+    /// </summary>
+    [JsonInclude]
+    [JsonPropertyName("category_id")]
+    public string CategoryID { get; set; }
```

Comment:
this field is needed if Category is Custom. OnePassword needs the categoryID to know which custom template was used.

Reply:
Understood.